tests/Rules/data/FooController.php

```diff
@@ -30,7 +31,20 @@ public function notExisting(): View
     }
 }
 
-class FooEmail extends Mailable
+class FooMailable extends Mailable
```

Comment:
The mailable and the mail message should use different views or the test is not going to test anything